db/migrate/20211110012408_create_articles.rb

```diff
@@ -0,0 +1,9 @@
+class CreateArticles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :description
```

Comment:
Why text?

Reply:
String will be limited to 255 characters and since we may expect much longer sentences inside of description, text data type was used here to allow for up to 30,000 characters.

Reply:
What is string to postgres db data type?

Reply:
It is referring to VARCHAR in postgres.

Reply:
Good.
Most rdbms uses varchar.